src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,166 @@
+				cancelMsg: i18nDict[ "exitscript-cancelMsg" ]
+			};
+			
+			wb.ready( $elm, componentName );
+		}
+
+	};
+
+	// Add your plugin event handler
+	$document.on( "click", selector, function( ev ) {
+		ev.preventDefault();
+		var elm = event.currentTarget,
+				$elm = $( elm ),
+				moDalId,
+				wrapper,
+				exitURLsearchParam,
+				exitURL,
+				exitParam = encodeURIComponent( "targetUrl=" + String( this.href ) );
+		var moDal = document.createDocumentFragment();
+		var tpl = document.createElement( "div" );
+		var settings = $.extend(
+		defaults, {}
+		);
+
+		if(this.hasAttribute("target")){ 
+		  exitParam = exitParam + encodeURIComponent( "&targetBlank=true" );
+		}
+			 
+		exitURL = settings.exitURL;
+			
+		if( !exitURL.includes( "targetUrl" ) ){ 
+	
+		if( !wb.getUrlParts( exitURL ).search ){
+			exitURL = wb.getUrlParts( exitURL ).href + "?" + exitParam;
+			exitParam = exitParam.split( '%26' ).join( '&' );
+	
+		} else { 
+				exitURLsearchParam = String(wb.getUrlParts( exitURL ).search );
+				exitURLsearchParam = exitURLsearchParam.replace("?","");		
+				exitParam =  exitURLsearchParam + "&" + exitParam;
+				exitParam = exitParam.split('%26').join('&');
+				//Build exitURL
+				exitURL = wb.getUrlParts( exitURL ).protocol + 
+				"//" + 
+				wb.getUrlParts( exitURL ).host + 
+				wb.getUrlParts( exitURL ).pathname  + 
+				"?" + 
+				exitParam;
+			} 
+		}
+			var defaultModadisplay = settings.displayModal !== false;
+			
+			// Set default properties
+			if ( defaultModadisplay ) {
+			   
+				if(this.hasAttribute("target")){
+					settings.targetBlank = "target='_blank'"
+				} 
+				if ( !settings.exitMsg ) {
+					settings.exitMsg = i18nDict.exitMsg
+				}
+				if ( !settings.yesMsg ) {
+					settings.yesMsg = i18nDict.yesMsg
+				}
+				if ( !settings.cancelMsg ) {
+					settings.cancelMsg = i18nDict.cancelMsg
+				}
+				 
+				tpl.innerHTML = "<section class='mfp-hide modal-dialog modal-content overlay-def'>" +
+					"<header class='modal-header'><h2 class='modal-title'>" + i18nDict.warning + "</h2></header>" +
+					"<div class='modal-body'>" +
+					"<p>" + settings.exitMsg + "</p>" +
+					"<ul class='list-inline text-center'>" +
+					"<li><button class='btn btn-primary popup-modal-dismiss pull-left' id='eCancel'>" + settings.cancelMsg + "</button></li>" +
+					"<li><a class='btn btn-default pull-right popup-modal-close' " + settings.targetBlank + " href='" + exitURL  + "' type='button'>" + settings.yesMsg + "</a></li>" +
+					"</ul></div></section>";
+			 
+				moDal.appendChild( tpl );
+				wrapper = moDal.firstChild;
+				wrapper.firstChild.id = wb.getId();
+				moDalId = wrapper.firstChild.id;
+				document.body.appendChild( wrapper.firstChild );
+				$document.trigger("open.wb-lbx", [
+					[{
+						src: "#" + moDalId,
+						type: "inline"
+					}],
+				 
+				true
+			 
+		]);					 
+ 
+                } else {
+					 
+				if ( this.hasAttribute("target") ) {
```

Comment:
Why to not consider the value of the attribute `target`? I think this case you are only looking for `target="_blank"`, is it?